sonarQuest-frontend/src/app/app-routing/app-routing.module.ts

```diff
@@ -19,6 +19,11 @@ const appRoutes: Routes = [
   {path: '', component: MainLayoutComponent, canActivateChild: [AuthenticationGuard], children: [
       {path: '', redirectTo: RoutingUrls.myAvatar, pathMatch: 'full'},
       {path: RoutingUrls.start, component: StartPageComponent},
+      {
```

Comment:
The `RoutingUrls.start` path is covered twice, so that your entry wouldn't be reached.

Reply:
This change is just a suggestion to use lazy loading and not meant as a proper "fix". Additionally lazy loading just one component doesnt make sense here, so unless all components are loaded lazily, I suggest to leave this PR open.

I try to add the missing routes unless someone else wants to :D